lib/token.js

```diff
@@ -1,6 +1,6 @@
 /* globals FRAME_ID */
-/*jshint maxcomplexity:15 */
-
+/* eslint-disable complexity */
```

Comment:
I think I'll use


```javascript
/* eslint-disable complexity, max-statements */
```

unless the `:0` suffix is how we're disabling it elsewhere